packages/services/rate-limit/src/index.ts

```diff
@@ -24,67 +23,28 @@ async function main() {
     });
   }
 
-  const server = await createServer({
-    name: 'rate-limit',
-    tracing: false,
-    log: {
-      level: env.log.level,
-    },
-  });
-
-  try {
-    const ctx = createRateLimiter({
-      logger: server.log,
-      rateLimitConfig: {
-        interval: env.limitCacheUpdateIntervalMs,
-      },
-      rateEstimator: env.hiveServices.usageEstimator,
-      emails: env.hiveServices.emails ?? undefined,
-      storage: {
-        connectionString: createConnectionString(env.postgres),
-      },
-    });
+  const logger = createLogger() as FastifyLoggerInstance;
 
-    await server.register(fastifyTRPCPlugin, {
-      prefix: '/trpc',
-      trpcOptions: {
-        router: rateLimitApiRouter,
-        createContext: () => ctx,
-      },
-    });
+  const app = createServerAdapter(rateLimitRouter);
+  const server = createServer(app);
 
+  try {
     registerShutdown({
-      logger: server.log,
+      logger,
       async onShutdown() {
-        await Promise.all([ctx.stop(), server.close()]);
-      },
-    });
-
-    server.route({
```

Comment:
Why did you remove these? they are used by the K8s and Docker for health and readiness

Reply:
they haven't been removed but extracted to an external router for simplifying the testing process in the future by using the router without the need to launch the server.

https://github.com/kamilkisiela/graphql-hive/pull/858/files#diff-783f53b46dea4ebd59badd5a393cefdb1b6bca12a68e67324e6ffdcbbf89473eR34-R55

Reply:
ok cool!